src/card_operations/card_operation_typedefs.h

```diff
@@ -44,6 +47,22 @@ typedef enum card_errors_type {
   CARD_OPERATION_DEFAULT_INVALID = 0xFF, /** Default invalid value */
 } card_error_type_e;
 
+typedef struct {
+  uint8_t acceptable_cards;
+  bool skip_card_removal;
```

Comment:
can add `family-id` as input too. helpful for situations like temp-pairing (also, card health check for not-paired cards if needed in the future).

Reply:
Added it as part of configuration to store expected family id and as part of response to return the card family id tapped